setup.py

```diff
@@ -10,29 +10,38 @@
             long_description += line
         else:
             break
+    
+extras = {
+    "testing": ["pytest==7.0.1"]
```

Comment:
Could you add a comment saying "pytest is pinned to 7.0.1 as this is last version for python 3.6"